src/platform/forms-system/src/js/review/submit-states/Pending.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Back from './Back';
+import ProgressButton from '../../components/ProgressButton';
+import PropTypes from 'prop-types';
+import { Column, Row } from 'platform/forms/components/common/grid';
+import PreSubmitSection from 'platform/forms/components/review/PreSubmitSection';
+
+export default function Pending(props) {
+  const { formConfig, onBack, onSubmit } = props;
+
+  return (
+    <>
+      <PreSubmitSection formConfig={formConfig} />
+      <Row classNames="form-progress-buttons">
+        <Column classNames="small-6 medium-5">
+          <Back onButtonClick={onBack} />
+        </Column>
+        <Column classNames="small-6 medium-5">
+          <ProgressButton
+            onButtonClick={onSubmit}
+            buttonText="Sending..."
+            disabled
+            buttonClass="usa-button-disabled"
+          />
```

Comment:
I'm not proposing a change here since this is just copy pasta, not new, but I'm wondering if we shouldn't have an `onButtonClick` for disabled buttons.

Reply:
oh dang, good call